stage0/stdlib/Lean/Compiler/IR/EmitC.c

```diff
@@ -585,6 +585,7 @@ lean_object* l_Lean_IR_EmitC_emitDeclAux___lambda__1(lean_object*, lean_object*,
 static lean_object* l_Nat_forM_loop___at_Lean_IR_EmitC_emitDeclAux___spec__1___closed__2;
 lean_object* l_Nat_forM_loop___at_Lean_IR_EmitC_emitDeclAux___spec__1___boxed(lean_object*, lean_object*, lean_object*, lean_object*, lean_object*);
 lean_object* l_Lean_IR_EmitC_toCType(lean_object*);
+static lean_object* l_Lean_IR_EmitC_emitDeclAux___closed__1;
```

Comment:
I'd love to move stage0 to a git submodule so we can prune it's history from time to time... hate seeing this code generated cruft mingled with real changes like this...

Reply:
Leo and I talked about doing this, but it would introduce more coordination headaches, especially around PRs. We'll gather some first experience with a Lake submodule (which shouldn't change as much) soon, then we can reevaluate this.